docs/getting_started.rst

```diff
@@ -7,9 +7,16 @@ Getting Started
 Features
 ========
 
-**niveristand** allows you to:
+**niveristand** has two major capabilities: system definition and real-time sequence scripting. NI recommends you use an editor with code completion, such as `Visual Studio Code <https://code.visualstudio.com/docs/languages/python/>`_, to make it easier to browse and use this code.
```

Comment:
suggestion: "niveristand has two major capabilities: system definition scripting, and real-time sequence scripting and deployment." Even though we elaborate just below this in the real-time sequence section, I think it is worthwhile to call out that the python api specifically supports deploying rt sequences.